frontend/lib/data-driven-onboarding/data-driven-onboarding.tsx

```diff
@@ -512,22 +512,22 @@ const ACTION_CARDS: ActionCardPropsCreator[] = [
   },
   function evictionFreeNy(data): ActionCardProps {
     const covidMessage = (
-      <Trans id="justfix.ddoEfnycCovidMessage">
+      <Trans id="justfix.ddoEvictionfreeCovidMessage">
         You can send a hardship declaration form to your landlord and local
         courts— putting your eviction case on hold until May 1st, 2021.
       </Trans>
     );
     return {
       title: li18n._(t`Protect yourself from eviction`),
-      priority: EFNYC_PRIORITY,
+      priority: EVICTIONFREE_PRIORITY,
       isRecommended:
         data.isRtcEligible && (data.numberOfEvictionsFromPortfolio || 0) > 0,
       indicators: [covidMessage],
       fallbackMessage: covidMessage,
       imageStaticURL: "frontend/img/ddo/judge.svg",
       cta: {
-        to: efnycURL(),
-        gaLabel: "efnyc",
+        to: evictionfreeURL(),
+        gaLabel: "evictionfree",
```

Comment:
Are we worried about this gaLabel (which I imagine could be a google analytics thing) getting renamed?